docs/uml-class-diagrams/admin/test/GpsSystemTests/GpsSystemTests.puml

```diff
@@ -2,7 +2,21 @@
 
 package edu.oakland.test.admin {
     class GpsSystemTests {
-
+        + void initDisplayGPSInterfaceNotNull()
+        + void initSatelliteNamesNotNull()
+        + void gpsSystemsConstructedSuccessfully()
+        + void useCase1RunSuccessfully()
+        + void uC1ScannerNotNull()
+        + void useCase2RunSuccessfully()
+        + void uC2ScannerNotNull()
+        + void satellietNamesNotNull()
```

Comment:
We can remove that if we're removing this as a method parameter in favor of just having that in the constructor.
```suggestion
```